README.md

```diff
@@ -78,7 +78,9 @@ or the configuration file can be specified by cli args as well.
 * `make run-mock-registry`: Mock registry. Entirely separate binary.
 * `make test`: Runs the test suite.
 * `make vendor`: Updates the dependencies
-* `make build`: Builds a docker container of the current source
+* `make dev-image`: Builds a docker container of the current source
```

Comment:
Should this be 'make build-image'   or should we change the target of the Makefile 'build-image' to be 'dev-image'?